### ISSUE TITLE
Adding "copied" feedback to Sample Code UI on API doc

### ISSUE THREAD

**Issue by galeyang** [flutter/flutter#24886]
In recent study, on master channel's API doc (e.g. [actions](https://master-docs-flutter-io.firebaseapp.com/flutter/material/AppBar/actions.html)) we observed that participant was unsure if the sample code was successfully copied to clipboard --- after clicking on the copy all sample code button, she pressed ctrl + C again.

To further improve the usability of Sample code UI, we suggest this change: after clicking on it, show the message: "_Successfully copied to clipboard_". A brief messages popping up at the bottom of the screen (i.e. toast/[snackbar](https://material.io/design/components/snackbars.html#usage) in material design) should be enough.

cc: @jayoung-lee @gspencergoog @InMatrix

**Comment by zoechi** [comment]
Issue moved to [flutter/website #1817](https://github.com/flutter/website/issues/1817) via [**ZenHub**](https://www.zenhub.com/)

**Comment by gspencergoog** [comment]
No, it's not a dartdoc change.  The "snippet" program is part of flutter [dev/snippets](https://github.com/flutter/flutter/tree/master/dev/snippets), and that's where the feedback snackbar would need to be added (or in [dev/docs/assets](https://github.com/flutter/flutter/tree/master/dev/docs/assets), really).

The complicated part here is that we'd need to code the snackbar from scratch.  Not too hard, but still a bit of work.

**Comment by Hixie** [comment]
Yeah probably.

**Comment by InMatrix** [comment]
Any updates on this? It looks a very simple feature.

**Comment by gspencergoog** [comment]
No, no update.  It's not too complicated, but I haven't had time to implement it.
